docs/DeveloperGuide.md

```diff
@@ -567,7 +577,7 @@ Priorities: High - must have; Medium - nice to have;  Low - unlikely to have.
 **MSS**
 
 1. User requests for help
-2. TAB displays a summary table of all the command usages as well as a link to an online user guide page
+2. TAB displays command guides.
```

Comment:
I think we can phrase this as "usage guide" that encompasses both the commands usage and user guide. What do you think? @Xiaoyunnn @lingshanng